assets/app/scripts/directives/logViewer.js

```diff
@@ -64,6 +64,52 @@ angular.module('openshiftConsole')
               logLinks.scrollTop();
             };
 
+            var bufferedLines = [];
```

Comment:
line.childNodes[0].childNodes[0].appendChild(…) isn't beautiful, but I think it would be limited to two things, the line number and the log text. What do you think?